.gitignore

```diff
@@ -311,3 +311,5 @@ dist
 ### OSX ###
 # General
 .DS_Store
+IaC/PnP/monitoring/aks/terraform.tfstate
```

Comment:
This will create a merge conflict with your other PR and can be removed from this one